ajax/libs/gzip-js/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "gzip-js",
+  "description": "GZIP in pure JavaScript (works in the browser)",
+  "filename": "gzip.min.js",
+  "keywords": [
+    "gzip",
+    "zip",
+    "pure javascript"
+  ],
+  "author": "T. Jameson Little <t.jameson.little@gmail.com>",
+  "homepage": "https://manifestwebdesign.github.io/angular-gridster",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/beatgammit/gzip-js.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/beatgammit/gzip-js.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
+        "files": [
+          "*.js"
```

Comment:
Let's use npm as the source of auto-update because there are releases on npm but no releases on GitHub.
```
  "npmName": "gzip-js",
  "npmFileMap": [
    {
      "basePath": "lib",
      "files": [
        "gzip*"
      ]
    }
  ]
```

Reply:
@sashberd Let's use npm
Because there are releases on npm but no releases on GitHub(Git tag).